src/data/qpf/multivariate/constructions/fix.lean

```diff
@@ -0,0 +1,296 @@
+/-
+Copyright (c) 2018 Jeremy Avigad. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Jeremy Avigad, Simon Hudon
+-/
+import data.pfunctor.multivariate.W
+import data.qpf.multivariate.basic
+universes u v
+
+/-!
+
+The initial algebra of a multivariate qpf is again a qpf.
+
+We define the corresponding polynomial functor by selecting:
```

Comment:
See the comments to `cofix.lean`. There is no `A` or `B` here either, and the same comments hold.